database/buildSchema/38b_application_list_filters.sql

```diff
@@ -0,0 +1,81 @@
+CREATE FUNCTION application_list_filter_applicant (applicant varchar, template_code varchar)
```

Comment:
I'm not entirely clear why these need to be functions, as opposed to just queries on existing tables. Can you briefly explain it?

Reply:
there is not distinct aggregate on graphQL queries. (I couldn't find any plugins that do it either)

To get a list of distinct values in graphQL we need to query all applications and compute distinct values on front end : /